src/util/datetime.ts

```diff
@@ -0,0 +1,9 @@
+import { DateTime } from 'luxon';
+
+export function formatDatetime(date: Date): string {
+  const parsedTime: DateTime = DateTime.fromJSDate(date);
+  if (!parsedTime.isValid) return undefined;
```

Comment:
is it better to return undefined or empty string `''`?

Reply:
I think would better '' coz it's a function that return string, updated in https://github.com/RR-Ardhika/murojaah/pull/8/commits/16c6dace217cd44350c7bdf7c1e455ea98bd237e, thank you